terraform/install.sh

```diff
@@ -333,12 +333,12 @@ parseArguments() {
       shift
       ;;
     -h|--help)
-      log "Deploy Stackdriver Sandbox to a GCP project"
+      log "Deploy Cloud Ops Sandbox to a GCP project"
       log ""
       log "options:"
-      log "-p|--project|--project-id     GCP project to deploy Stackdriver Sandbox to"
+      log "-p|--project|--project-id     GCP project to deploy Cloud Ops Sandbox to"
       log "-v|--verbose                  print commands as they run (set -x)"
-      log "--skip-workspace-prompt       Don't pause for Stackdriver workspace set up"
+      log "--skip-workspace-prompt       Don't pause for Cloud Ops workspace set up"
```

Comment:
I think here it should say 'Don't pause for Cloud Monitoring Workspace set up'

Reply:
Yeah, it should be.